flow/packet-headers/rsvp.yaml

```diff
@@ -1261,13 +1289,13 @@ components:
             If OTG is unable to generate a value the default value must be used.
           type: integer
           format: uint32
-          maximum: 256
+          maximum: 255
```

Comment:
max can be removed since this is not user input

Reply:
Done.